.github/workflows/build-design-system.yml

```
@@ -0,0 +1,34 @@
+  pull_request:
+    paths:
+      - "packages/design-system/src/tokens/**"
+
+jobs:
+  build-design-system:
+    runs-on: ubuntu-latest
+
+    steps:
+    # Step 1: Checkout the repository
+    - name: Checkout repository
+      uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)